extensions/wikia/MyHome/SpecialWikiActivity.class.php

```diff
@@ -9,150 +9,183 @@ class SpecialWikiActivity extends UnlistedSpecialPage {
 	private $feedSelected;
 
 	function __construct() {
-		parent::__construct('WikiActivity', '' /* no restriction */, true /* listed */);
+		parent::__construct( 'WikiActivity', '' /* no restriction */, true /* listed */ );
 	}
 
-	function execute($par) {
-		wfProfileIn(__METHOD__);
+	function execute( $par ) {
+		wfProfileIn( __METHOD__ );
 		global $wgOut, $wgUser, $wgBlankImgUrl, $wgEditPageFrameOptions;
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Refactor this code to rely on RequestContext instead of context-dependent globals. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=wikia-php-rules%3AContextualWgVariableUsage)
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Refactor this code to rely on RequestContext instead of context-dependent globals. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=wikia-php-rules%3AContextualWgVariableUsage)